src/common/low_precision_transformations/src/broadcast.cpp

```diff
@@ -5,40 +5,30 @@
 #include "low_precision/broadcast.hpp"
 
 #include <memory>
-#include "openvino/opsets/opset1_decl.hpp"
-#include "openvino/opsets/opset3_decl.hpp"
-#include "openvino/pass/pattern/op/or.hpp"
-#include "openvino/pass/pattern/op/wrap_type.hpp"
-#include "low_precision/network_helper.hpp"
 
 #include "itt.hpp"
+#include "low_precision/network_helper.hpp"
 #include "openvino/op/broadcast.hpp"
+#include "openvino/pass/pattern/op/wrap_type.hpp"
 
 using namespace ov::pass::low_precision;
 
 BroadcastTransformation::BroadcastTransformation(const Params& params) : TransparentBaseTransformation(params) {
     MATCHER_SCOPE(BroadcastTransformation);
-    auto broadcast1 = pattern::wrap_type<ov::opset1::Broadcast>({
-        pattern::wrap_type<ov::opset1::Multiply>(),
-        ov::pass::pattern::any_input(),
-        ov::pass::pattern::any_input() });
-
-    auto broadcast3 = pattern::wrap_type<ov::opset3::Broadcast>({
-        pattern::wrap_type<ov::opset1::Multiply>(),
-        ov::pass::pattern::any_input(),
-        ov::pass::pattern::any_input() });
-
-    const auto matcher = std::make_shared<ov::pass::pattern::op::Or>(ov::OutputVector{ broadcast1, broadcast3 });
+    using namespace ov::pass::pattern;
+    auto mul = wrap_type<ov::op::v1::Multiply>();
+    auto matcher = wrap_type<ov::op::v3::Broadcast, ov::op::v1::Broadcast>({mul, any_input(), any_input()}) |
```

Comment:
```suggestion
    auto matcher = wrap_type<ov::op::v3::Broadcast, ov::op::v1::Broadcast>({mul, any_input(), any_input()}) ||
```
I suggest to avoid using bitwise operation is such context

Reply:
operator| is specially overloaded for these types:
https://github.com/openvinotoolkit/openvino/blob/47aac92e0610eb1b333cc6e8a407d42859dff261/src/core/include/openvino/pass/pattern/op/or.hpp#L27-L28
So semantically it is not bitwise, and its usage in such context is legal :)